modules/client-configuration/pages/autoinst-reinstall.adoc

```diff
@@ -0,0 +1,18 @@
+[[reinstalling-registered-systems]]
+= Reinstalling Registered Systems
+
+The in-place reinstallation is a purely local mechanism. There is therefore no need for PXE booting capabilities on the client.
+
+To reinstall a registered client in-place, you must first define an autoinstallation distribution and an autoinstallation profile.
```

Comment:
```suggestion
To reinstall a registered client in-place, you must have defined an autoinstallation distribution and an autoinstallation profile.
```

Reply:
I have a problem with that phrasing you suggest. I would like to describe steps in a procedure: first do this, then do that.

But if you say that something should have already been done in a prior stage, that's a problem, because we have not even explained yet how to do it!

I would like to keep the "first".

Reply:
https://github.com/uyuni-project/uyuni-docs/wiki/wug#accessibility